ios/XMTPModule.swift

```diff
@@ -2457,11 +2460,14 @@ public class XMTPModule: Module {
 		}
 	}
 
-	func createApiClient(env: String)
+	func createApiClient(env: String, customLocalUrl: String? = nil)
 		-> XMTP.ClientOptions.Api
 	{
 		switch env {
 		case "local":
+			if let customLocalUrl = customLocalUrl, !customLocalUrl.isEmpty {
+				XMTP.XMTPEnvironment.customLocalAddress = customLocalUrl
+			}
```

Comment:
Maybe totally wrong here but shouldn't this be: 

```suggestion
			if let customLocalUrl = customLocalUrl, !customLocalUrl.isEmpty {
                              return XMTP.ClientOptions.Api(
	                              env: XMTP.XMTPEnvironment.custom(localAddress: customLocalUrl),
	                              isSecure: false
                              )
			}
```